Add database seeding to `User` table

pedro-m-dietrich opened the pull request:
The seeding creates 50 users in the database.

pedro-m-dietrich committed:
Add database seeding to `User` table

The seeding creates 50 users in the database.

alanraso commented on db_seeding/user_seed.ts:
```diff
@@ -0,0 +1,35 @@
+
+initializeDatabaseInstance();
+
+async function main() {
+  console.log('Seeding database users...');
+
+  for (let i = 1; i <= 50; i++) {
+    const name: string = 'User ' + i;
+    const email: string = 'user' + i + '@seeded.com';
+
+    await prisma.user.upsert({
+      where: { email: email },
+      update: {},
```
Aqui para um desempenho melhor no seu ambiente local, acho que seria interessante você adicionar todos os usuários fazendo apenas uma query pro banco. Dá pra adicionar um array nos métodos do Prisma, se não me engano.

pedro-m-dietrich committed:
Update seeding to create users in a single query

Intead of using `upsert()` multiple times in a loop, the users are
stored in an array to then be inserted in the database using
`createMany()` with the `skipDuplicates` option enabled.
